src/citrine/informatics/predictors.py

```diff
@@ -81,7 +81,7 @@ class SimpleMLPredictor(Serializable['SimplePredictor'], Predictor):
     latent_variables = properties.List(properties.Object(Descriptor), 'config.latent_variables')
     training_data = properties.Object(DataTable, 'config.training_data')
     typ = properties.String('config.type', default='Simple', deserializable=False)
-    status = properties.String('status', serializable=False)
+    status = properties.Optional(properties.String(), 'status', serializable=False)
```

Comment:
I made `status` optional for all non-graph predictors so that they could be embedded and not have a status. We could also give all embedded predictors a status like "EMBEDDED". I'm not sure what would make more sense from the user's point of view